client/assets/sass/_explorer.sass

```diff
@@ -62,3 +62,6 @@
 .card-view-pf
   .card
     cursor: pointer
+
+.service-explorer-row
```

Comment:
Lets try to keep the names explorer agnostic, rather service-explorer, prefer `explorer-row`